apps/lib/apps.c

```diff
@@ -1024,6 +1023,7 @@ int load_key_certs_crls(const char *uri, int maybe_stdin,
 
  end:
     OSSL_STORE_close(ctx);
+    BIO_free(bio);
```

Comment:
@t8m - does this change look ok to you? Took me a while to figure out .

Reply:
It would be nice (but not required for this PR) to add a note to OSSL_STORE_attach, the semantics of the bio attachment. I.E., that it bumps the reference count of the bio. Would you like to do that? Or I can open a separate PR.

Reply:
Ok will add a commit to this PR..

Reply:
More concerning is the leak was not picked up..